src/appleseed-max-impl/appleseedrenderer/appleseedrendererparamdlg.cpp

```diff
@@ -928,7 +459,49 @@ AppleseedRendererParamDlg::AppleseedRendererParamDlg(
     RendererSettings&       settings,
     AppleseedRenderer*      renderer)
   : impl(new Impl(rend_params, in_progress, settings, renderer))
+  , m_renderer(renderer)
+  , m_pmap_output(nullptr)
+  , m_pmap_image_sampling(nullptr)
+  , m_pmap_lighting(nullptr)
+  , m_pmap_system(nullptr)
 {
+   m_pmap_output = CreateRParamMap2(
+        0,
+        renderer->GetParamBlock(0),
+        rend_params,
+        g_module,
+        MAKEINTRESOURCE(IDD_FORMVIEW_RENDERERPARAMS_OUTPUT),
+        L"Output",
+        0,
+        new OutputParamMapDlgProc(renderer->GetParamBlock(0)));
+
+   m_pmap_image_sampling = CreateRParamMap2(
```

Comment:
Can you order these calls (as well as all related stuff such as member variable declarations and initializations, etc.) according to the order of the panels in the UI? I.e. image sampling stuff first, etc. It makes things easier to find.

Reply:
I think they are in order, aren't they?  
m_pmap_output
m_pmap_image_sampling
m_pmap_lighting
m_pmap_system

![image](https://user-images.githubusercontent.com/1850877/40881283-79533bc6-6690-11e8-9951-40ea541466d8.png)